cmake/config.h.in

```diff
@@ -0,0 +1,151 @@
+/* #define to the attribute for default visibility. */
```

Comment:
`config.h.in` from the root of the project can be ealisy re-used (done it in one of my projects)

Reply:
config.h.in seems to be a generated file after running bootstrap script.
this is as close to it as i can get.